package.json

```diff
@@ -49,12 +49,12 @@
     "typescript": "^4.3.3"
   },
   "dependencies": {
-    "@uniswap/universal-router": "1.4.3",
     "@uniswap/permit2-sdk": "^1.2.0",
-    "@uniswap/router-sdk": "^1.4.0",
-    "@uniswap/sdk-core": "^3.1.0",
-    "@uniswap/v2-sdk": "^3.0.1",
-    "@uniswap/v3-sdk": "^3.9.0",
+    "@uniswap/router-sdk": "^1.6.0",
+    "@uniswap/sdk-core": "^4.0.0",
+    "@uniswap/universal-router": "1.4.3",
+    "@uniswap/v2-sdk": "^3.2.0",
+    "@uniswap/v3-sdk": "^3.10.0",
```

Comment:
we need to update all the other packages too, because they *also* depend on `sdk-core`. all the versions of `sdk-core` used in this package need to match even if they're used by a dependency